test/test_owners.json

```diff
@@ -39,7 +39,7 @@
  "cli": [
   "Kubectl"
  ],
- "cluster-lifecycle": [
+ "cloud-provider-gcp": [
   "GKE node",
   "Garbage",
   "Nodes",
```

Comment:
Is this still right?

Reply:
Ah I see, this is based on test/test_owners.csv

Reply:
looks like test_owners* is tech debt that is bound to both k/k and test-infra ATM.